Zend/zend_exceptions.stub.php

```diff
@@ -24,14 +24,11 @@ class Exception implements Throwable
 {
     /** @var string */
     protected $message = "";
```

Comment:
I'm not sure here how far we can go here... I remember Symfony had some issues with Exception property types, so I'm wondering what to do with `$message` and `$code`

Reply:
Ping @nicolas-grekas in case you missed this message and there's something that Symfony needs to do related to this. Thanks!

Reply:
`string|Stringable` would be the most BC type description. We do use a `Stringable` object here, and it works great (the cast to string happens when calling `getMessage()`)

Reply:
Yeah, I think we need to leave at least $message and $code alone, they are often used in non-standard ways (including by PHP itself -- see PDO a string codes)

Reply:
Thanks Javier and Nicolas for your input, but based on Nikita's feedback, I'll leave these properties alone :)

Reply:
maybe a comment could be added somewhere in the code to remind a future contributor that these have no type on purpose?